### PR TITLE
Overriding styles for any element inside a default description

### LINKED ISSUE #3457: Weird description effect on public dashboard
screen shot 2015-05-04 at 10 19 16

marias.cartodb.com

### PR DESCRIPTION
**Pull request by xavijam**
That means, h1, h2, h3... elements won't be rendered as big(...small) titles, they will have all the same style.

screen shot 2015-05-04 at 15 36 19
screen shot 2015-05-04 at 15 44 54
screen shot 2015-05-04 at 15 36 39
screen shot 2015-05-04 at 15 36 31

Fixes #3457

### CODE REVIEW

**Comment by xavijam** [comment]
This solution was agreed with @saleiva, what do you think @alonsogarciapablo?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3427/))

File: app/assets/stylesheets/new_common/default-description.css.scss [hunk 1]
```diff
@@ -13,6 +13,12 @@
   overflow-wrap: break-word;
   position: relative;
 }
+.DefaultDescription > * {
+  margin: 0;
+  padding: 0;
+  font-weight: $sFontWeight-normal;
+  font-size: $sFontSize-normal
```
Comment (alonsogarciapablo) [inline app/assets/stylesheets/new_common/default-description.css.scss]: You're missing a semicolon at the end of this line.

**Comment by alonsogarciapablo** [comment]
Left a minor comment. Other than that... :+1:

File: app/assets/stylesheets/new_common/default-description.css.scss (hunk 1 above)
Comment (xavijam) [inline app/assets/stylesheets/new_common/default-description.css.scss]: :dancer: thanks

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3428/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3430/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3431/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3434/))

**Pull request merged**